interface/main/tabs/templates/tabs_template.php

```diff
@@ -18,7 +18,7 @@
         <!-- ko  foreach: tabsList -->
             <div class="tabSpan bgcolor2" data-bind="click: tabClicked, css: {tabNotchosen: !visible()}">
                 <span class="tabTitle" data-bind="text: title, click: tabClicked, css: {tabHidden: !visible()}"></span>
-                <span class="fa fa-fw fa-refresh" data-bind="click: tabRefresh"></span>
+                <span class="fa fa-fw fa-refresh" data-bind="click: tabRefresh, class: spinner"></span>
```

Comment:
> just make fa-sm, fa-1x or remove completely and spinner will default to title font size. no need to mess with sizing it!!!

@sjpadgett I removed the spinner now the refresh button itself is spinner. Although doesn't spin while refresh.So for that i will make another PR becuze this PR has already so many changes 👍